definitions/marts/looker_studio_marts/npq/ls_npq_acceptance_reversal.sqlx

```diff
@@ -17,27 +17,33 @@ config {
 
 WITH acceptance_reversion AS (
   SELECT
-     status_changes.entity_id AS application_id
-    ,status_changes.ecf_id AS application_ecf_id
-    ,COUNT(DISTINCT request_uuid) AS accepted_reversal_count
+     application_id
+    ,application_ecf_id
+    ,SUM(accepted_reversal_count) AS accepted_reversal_count

```

Comment:
Could this sum still contain duplicates if there were overlaps in the UNION?

Reply:
By doing the grouping on the result of the UNION it returns a single row per application ultimately. This was implemented after testing as there were duplicates cause by applications appearing in both halves of the union.